great_expectations/rule_based_profiler/types/data_assistant_result/data_assistant_result.py

```diff
@@ -1854,26 +1864,31 @@ def _chart_column_values(
                     List[ColumnDataFrame],
                     str,
                     alt.StandardType,
+                    bool,
+                    Optional[str],
                 ],
                 alt.VConcatChart,
             ]
         ] = None
-        display_chart: Optional[alt.VConcatChart] = None
 
-        if metric_name == "column_distinct_values_count":
+        subtitle: Optional[str] = None
+        if metric_name:
             if plot_mode is PlotMode.PRESCRIPTIVE:
                 plot_impl = (
                     self.get_interactive_detail_expect_column_values_to_be_between_chart
                 )
+                subtitle = expectation_type
```

Comment:
Expectation name subtitle is only relevant for PRESCRIPTIVE (`expectations_and_metrics`)